test/e2e/common/expansion.go

```diff
@@ -29,7 +29,6 @@ import (
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	. "github.com/onsi/ginkgo"
```

Comment:
We could also remove the `.` dot imports here ?

Reply:
Yeah, we could with another PR for keeping this PR scope simple and clean.